bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/storeconfigexporter/StoreConfigExporterImpl.java

```diff
@@ -88,6 +95,18 @@ public String getMethod() {
         return method.toString();
     }
 
+    public String getHttpHeaders() {
+        ObjectMapper mapper = new ObjectMapper();
+        ObjectNode objectNode = mapper.createObjectNode();
+        httpHeaders.entrySet().stream().forEach(entry -> objectNode.put(entry.getKey(), entry.getValue()));
+        try {
+            return mapper.writeValueAsString(objectNode);
+        } catch (JsonProcessingException e) {
+            LOGGER.error(e.getMessage(), e);
+            return "";
```

Comment:
In Venia you parse this value without null or empty checks, see 

https://github.com/adobe/aem-cif-guides-venia/blob/2801fdc943669572cafe3c67cbd3628a96485896/ui.frontend/src/main/components/App/App.js#L48

If you return `""` in case of an error, the `JSON.parse` in Venia will fail. How about instead returning an empty object `{}`?

Reply:
Thanks for catching this, I'll change it.